sdk/cmake/init-section.lds

```diff
@@ -8,5 +8,4 @@ SECTIONS
     __init_end__ = . ;
   }
 }
-/*INSERT AFTER .reloc;*//*This is what we actually want to do, but do not dare due to LD bug ROSBE-154*/
-INSERT AFTER .rsrc; /*This is what we do instead to at least fix CORE-14683*/
+INSERT BEFORE .reloc; /* .reloc is always at the end */
```

Comment:
In our MSVC binaries and in Windows drivers, I see the following order: `INIT`, `.rsrc`, `.reloc`. Unless there are other limitations, I suggest to unify the behavior

Reply:
.rsrc is non-discardable for ntoskrnl.

Reply:
I just don't like that we would have different layout for MSVC and GCC builds

Reply:
Done